src/components/Login/Login.js

```diff
@@ -0,0 +1,62 @@
+import React, {Component} from 'react';
+import RaisedButton from 'material-ui/RaisedButton';
+import TextField from 'material-ui/TextField';
+import AuthService from '../../services/AuthService';
+
+class Login extends Component {
+    constructor(props) {
+        super(props);
+        this.state={username: '', password: ''}
+        this.handleSubmit = this.handleSubmit.bind(this);
+        this.onUserNameChange = this.onUserNameChange.bind(this);
+        this.onPasswordChange = this.onPasswordChange.bind(this);
+    }
+
+    handleSubmit(event) {
+        event.preventDefault();
+        AuthService.login(this.state.username, this.state.password)
+            .then(response => {
+                sessionStorage.setItem("jwt", response.token);
+            })
+            .catch(error => {
+                throw(error);
+            });
+    }
+
+    onUserNameChange(event) {
+        this.setState({username: event.target.value});
+    }
+
+    onPasswordChange(event) {
+        this.setState({password: event.target.value});
```

Comment:
I dont know if we want to save the password to state, we should probably just submit it with the form and not keep it in JS at all

Reply:
like in `handleSubmit` we serialize the form and pass the values to AuthService

Reply:
In: 2203a0a